protocol-units/da/movement/protocol/light-node/src/sequencer.rs

```diff
@@ -140,16 +140,17 @@ where
 	V: VerifierOperations<DaBlob<C>, DaBlob<C>>,
 {
 	async fn tick_build_blocks(&self, sender: Sender<Block>) -> Result<(), anyhow::Error> {
+		info!(target: "movement_timing", "tick_build_blocks");
 		let memseq = self.memseq.clone();
 
 		// this has an internal timeout based on its building time
 		// so in the worst case scenario we will roughly double the internal timeout
 		let uid = LOGGING_UID.fetch_add(1, std::sync::atomic::Ordering::SeqCst);
-		debug!(target: "movement_timing", uid = %uid, "waiting_for_next_block",);
+		info!(target: "movement_timing", tick = &uid, "CALLED tick_build_blocks tick");
```

Comment:
Can't we just run the binary with `RUST_LOG=debug`?

Reply:
We discussed with Radu, for now too many debug logs to do will be noisy.